src/std/math.ab

```diff
@@ -3,3 +3,27 @@ pub fun sum(list: [Num]): Num {
     return unsafe $echo "{list}" | awk '\{s=0; for (i=1; i<=NF; i++) s+=\$i; print s}'$ as Num
 }
 
+/// Returns the number rounded to the nearest integer
+#[allow_absurd_cast]
+pub fun round(number: Num): Num {
+    return unsafe $printf "%0.f" "{number}"$ as Num
+}
+
+/// Returns the largest integer less than or equal to the number
+#[allow_absurd_cast]
+pub fun floor(number: Num): Num {
+    return unsafe $echo "{number}" | awk '\{printf "%d", (\$1 < 0 ? int(\$1) - 1 : int(\$1))}'$ as Num
```

Comment:
uhhh, what? im pretty sure bash doesnt have a ternary operator. is this `awk`'s syntax? on which version is it supported? does it work the same on most platforms?

Reply:
Considering the single quotes around it, that's indeed a part of awk's script. In GNU manual these are called "conditional expressions".

https://www.gnu.org/software/gawk/manual/gawk.html#Conditional-Exp

Reply:
The following statement suggests that ternary operator (or "conditional expressions") is universally supported in awk:

> As a minor gawk extension, a statement that uses ‘?:’ can be continued simply by putting a newline after either character.

Reply:
> on which version is it supported?

The ternary operator ?: in awk was first introduced with the release of [System V Release 3.1](https://en.wikipedia.org/wiki/UNIX_System_V) in 1987. This feature is documented in the [awk changelog](https://www.gnu.org/software/gawk/manual/html_node/V7_002fSVR3_002e1.html)

>The conditional expression using the ternary operator ‘?:’ (see [Conditional Expressions](https://www.gnu.org/software/gawk/manual/html_node/Conditional-Exp.html)) 

So i would say it's widely supported.